### ISSUE TITLE
Extension causes high cpu load

### ISSUE THREAD

**Issue by futuri** [mrmlnc/vscode-postcss-sorting#87]
- Issue Type: `Performance`
- Extension Name: `vscode-postcss-sorting`
- Extension Version: `3.0.1`
- OS Version: `Linux x64 5.7.0-1-amd64`
- VSCode version: `1.47.0-insider`

[mrmlnc.vscode-postcss-sorting-unresponsive.cpuprofile.txt](https://github.com/mrmlnc/vscode-postcss-sorting/files/4885979/mrmlnc.vscode-postcss-sorting-unresponsive.cpuprofile.txt)

Find more details here: https://github.com/microsoft/vscode/wiki/Explain-extension-causes-high-cpu-load

**Comment by SiteSplat** [comment]
yup doesn't work! high CPU load and nothing happens!!! so annoying. why publishing something doesn't work?

**Comment by neuropass** [comment]
does NOT work!! CPU goes nuts and nothing happens. Why publishing this if it doesn't work?
